tests/utils/test_cases/conan_client.py

```
@@ -34,4 +52,4 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         os.chdir(cls._old_cwd)
-        shutil.rmtree(cls._working_dir)
\ No newline at end of file
+        shutil.rmtree(cls._working_dir)
```

I saw random errors on Windows. Should we ignore the errors when removing the working dir?

---

What kind of random errors? Can you link to any?

---

most related to busy working dir on Windows, same from Conan tests. I saw those errors on Conan's Jenkins jobs.

but as you are using this statement for tests only, I see no problem here.

---

I thought you mean errors/warnings in this repo... for Conan itself this will always raise because the connection to the database is never closed (here I'm closing it), and maybe some tools may break in the future if they leave some file open or so, but I think we should keep this here and we will see what to do if something fails

---

I agree.